site/content/blog/summer-updates-2023.md

```diff
@@ -0,0 +1,61 @@
+---
+title: What We Shipped in Jul-Aug 2023
+authors: ['ola-rubaj']
+date: 2023-09-22
+---
+
+Hey everyone! 👋 Summer has been in full swing, and while I've managed to catch some vacation vibes, I've also been deep into code. I'm super excited to share some of the latest updates and features we've rolled out over the past two months. Let's dive in:
+
+## 🌷 [Flowershow](https://flowershow.app/)
+
+1. **CLI is old news**: the Flowershow CLI has been deprecated in favor of our new [Flowershow Obsidian plugin](https://github.com/datopian/obsidian-flowershow).
+
+2. **Self-publishing made even easier**: I wrote a [self-publish tutorial](https://flowershow.app/docs/publish-howto) to guide you through using our Obsidian plugin for publishing your digital garden.
+
+3. **Cloud-publishing MVP**: The first version of our Flowershow Cloud which aims to make publishing your digital garden notes a breeze! [Check out the overview](https://flowershow.app#cloud-publish) and stay tuned 📻!
+
+4. **Hydration errors from Obsidian transclusions/note embeddings are fixed**: For now, note transclusions will convert to regular links, but stay tuned — support for note embeds may be on the horizon. [Learn more](https://github.com/datopian/flowershow/issues/545)
+
+5. **New Obsidian tags list format support**: I added support for Obsidian's new tag list format, so now your notes can be even more organized. [Learn more](https://github.com/datopian/flowershow/issues/543)
+
+## 🗂️ [MarkdownDB](https://github.com/datopian/markdowndb)
```

Comment:
One tip: generally linking headings breaks the "link to heading" feature of flowershow and is also kind of poor UX (i don't expect headings to link elsewhere).

So one small change: just put the link immediate below the heading e.g.

```
## 🗂️  MarkdownDB

https://github.com/datopian/markdowndb
```